src/main/java/edu/kis/powp/jobs2d/observer/Subscriber.java

```diff
@@ -0,0 +1,5 @@
+package edu.kis.powp.jobs2d.observer;
+
+public interface Subscriber {
```

Comment:
Similarly, the subscriber is already implemented and available (through dependent libs)